src/test/resources/post-response.json

```diff
@@ -1,5 +1,6 @@
 {
   "post_number": 22,
   "topic_id": 422,
+  "id": 12345,
   "topic_slug": "stone-test-topic"
 }
```

Comment:
Should we add setPostResponseData() ?

Reply:
I did that, but there's no usages yet and I think this file still needs to exist for the httpclientsimulator. Let me know if you disagree.

Reply:
You are right.  It is the default response simulated post responses.